chrome_manifest_v3/views/gmailView.js

```
@@ -1,538 +1,563 @@
+    : '';
+
+  let url_with_filename = function (url_in = '', var_in = '') {
+    return self.parent.url_add_var(
+      url_in,
+      self.parent.UNIQUE_URI_VAR + '=/' + var_in
+    );
+  };
+
+  let email_raw_md = function (name = '', email = '') {
+    let raw = '',
+      md = '';
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)